### DISCUSSION TITLE
My stats are wrong

### DISCUSSION THREAD

**Discussion by FRJess** [anuraghazra/github-readme-stats#2503] [Q&A] [answered]
Hi all and thanks for the amazing tools!

I think my stats are wrong and I can't understand why. It shows more than 1k commits in 2023 but it can't be. Also, I cannot understand how I could be rank A+. I'm just a baby dev ☺️

Here's my profile if you could check for the README file: https://github.com/FRJess

**Comment by mkredzel** [comment, marked as answer]
Hi @FRJess,

There is an issue already opened regarding this. Instead of displaying commits starting from 2023, it displays the number of commits made in the last 365 days. There is already a PR with a solution that is awaiting to be merged.

You can always fork the repo and deploy it on your [Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance) not to wait for PR to be merged.

- Discussion: https://github.com/anuraghazra/github-readme-stats/discussions/2498#discussioncomment-4876700
- Issue: https://github.com/anuraghazra/github-readme-stats/issues/2501

**Reply by Zo-Bro-23** [reply]
Also, the current ranking system is slightly confusing. A+ is not the highest rank; instead, A++, S, S+, and S++ are all above A+! There are several issues and prs that are working towards changing this, and we are considering changing the ranking system. Hope this answers your question!

**Comment by FRJess** [comment]
Thanks @mkredzel and @Zo-Bro-23 for your answers and explanations. It's all more clear now.
